contracts/voting-verifier/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+## [Unreleased](https://github.com/axelarnetwork/axelar-amplifier/tree/HEAD)
+
+[Full Changelog](https://github.com/axelarnetwork/axelar-amplifier/compare/voting-verifier-v1.2.0..HEAD)
+
+## [v1.2.0](https://github.com/axelarnetwork/axelar-amplifier/tree/voting-verifier-v1.2.0)
+
+[Full Changelog](https://github.com/axelarnetwork/axelar-amplifier/compare/voting-verifier-v1.1.0..voting-verifier-v1.2.0)
+
```

Comment:
could you manually add the entry specifying the Solana address validation was added? The script probably didn't catch that because it was added to a dependency and not to the contract crate directly,